src/lib/lti/tool.js

```diff
@@ -285,7 +285,7 @@ lti.onDeepLinking(async (token, req, res) => {
 const setup = async () => {
   Log('lib/tool/lti.setup(): started');
   try{
-    await lti.deploy({ serverless: true });
+    await lti.deploy({ serverless: true, debug: true });
```

Comment:
debug, should be configurable from our own config

Reply:
fixed